src/services/formatting/formatting.ts

```diff
@@ -1118,23 +1118,37 @@ namespace ts.formatting {
     }
 
     /**
-     * @returns -1 iff the position is not in a multi-line comment.
+     * Gets the indentation level of the multi-line comment enclosing position,
+     * and a negative value if the position is not in a multi-line comment.
      */
-    export function getIndentationOfEnclosingMultiLineComment(sourceFile: SourceFile, position: number): number {
-        const token = getTokenAtPosition(sourceFile, position, /*includeJsDocComment*/ false);
-        const leadingCommentRanges = getLeadingCommentRangesOfNode(token, sourceFile);
-        if (leadingCommentRanges) {
-            loop: for (const range of leadingCommentRanges) {
+    export function getIndentationOfEnclosingMultiLineComment(sourceFile: SourceFile, position: number, options: EditorSettings): number {
+        const range = getRangeOfEnclosingComment(sourceFile, position, /*onlyMultiLine*/ true);
+        if (range) {

```

Comment:
Just in-lined the helper since it's only called once. Additionally, the logic has changed because I discovered an issue with determining the columns when we have text which is non-whitespace (because determining the number of graphemes in a string is [non-trivial](https://github.com/orling/grapheme-splitter).